### DISCUSSION TITLE
ERROR: Could not create SSL/TLS secure channel.

### DISCUSSION THREAD

**Discussion by StaSiii** [fedorovvl/tso_client#179] [Q&A] [answered]
Something like this happened yesterday to me and one of my friends:

![Untitled](https://user-images.githubusercontent.com/117428815/221352002-0c208e70-bc8e-482e-8afe-f8171e632257.jpg)

And we can't play on the client. On unity, logging by the site, it is all okay.
I noticed, that only me and that friend have this issue, Maybe because of that we both have windows 7, friends with windows 10 can play.

edit: The same issue is on windows 8.1.

**Comment by Maksimov112358** [comment]
Та же самая ошибка. На вин7.

**Reply by jeffme1967** [reply]
hi ! the same error for me : do you have a tip to solve it ?
My OS : Windows 7 32 bits...
Thanks for answer !!
:-)

**Comment by fedorovvl** [comment, marked as answer]
pls check latest release

**Comment by Maksimov112358** [comment]
It works! Glory to Fedorov! :)
